client/state/index.js

```diff
@@ -120,6 +121,7 @@ export function createReduxStore( initialState = {} ) {
 	const middlewares = [
 		thunkMiddleware,
 		noticesMiddleware,
+		happychatMiddleware,
```

Comment:
I think it should be `isBrowser && happychatMiddleware` since we didn't seem to need it for SSR.